nilearn/maskers/base_masker.py

```diff
@@ -298,15 +301,16 @@ def transform(self, imgs, confounds=None, sample_mask=None):
             imgs, confounds=all_confounds, sample_mask=sample_mask
         )
 
+    @rename_parameters(replacement_params={"X": "imgs"}, end_version="0.13.2")
     @fill_doc
     def fit_transform(
```

Comment:
Just to make sure I am getting this right, wouldn't this also change the input parameter name from `X` ->` imgs` in `fit_transform` for all classes that inherit BaseMasker (meaning all Nifti*Maskers)?

I did not expect that just from reading the changelog entry. Maybe that should also be reflected there.

Reply:
ha yeah I forgot that one.

Reply:
though in practice only the NiftiMasker does not have its own fit_transform method and therefore relies on that of the parent class